.circleci/config.yml

```
@@ -135,13 +138,18 @@ jobs:
             - run:
                 name: Run UI Tests
                 command: "PATH=/usr/local/node-v10.24.1-linux-x64/bin:$PATH $XDMOD_SRC_DIR/tests/ui/runtests.sh --headless --log-junit ~/phpunit"
+            - run:
+                name: Ensure that no unexpected Apache errors were generated
+                command: test ! -e /var/log/php-fpm/www-error.log
```

Might be helpful to print the errors if there are any:
```suggestion
                command: >
                    if [ -e /var/log/php-fpm/www-error.log ]; then
                        cat /var/log/php-fpm/www-error.log;
                        false;
                    fi
```

---

This file is copied to the build artifacts (and hence the contents appear in the circle ci interface) so no need to redundantly output the errors twice.